frontend/src/components/logo.tsx

```diff
@@ -0,0 +1,14 @@
+import logoSymbolBlack from "/logo/ZaneOps-SYMBOL-BLACK.svg";
+import logoSymbolWhite from "/logo/ZaneOps-SYMBOL-WHITE.svg";
+
+export function Logo({ className = "hidden" }: { className: string }) {
```

Comment:
```suggestion
export function Logo({ className }: { className?: string }) {
```

Make this prop optional.